src/main/java/com/mraof/minestuck/MinestuckConfig.java

```diff
@@ -222,7 +222,7 @@ private Common(ForgeConfigSpec.Builder builder)
 		canBreakGates = SERVER_BUILDER.comment("Lets gates be destroyed by explosions. Turning this off will make gates use the same explosion resistance as bedrock.")
 				.define("canBreakGates",true);
 		disableGiclops = SERVER_BUILDER.comment("Right now, the giclops pathfinding is currently causing huge amounts of lag due to their size. This option is a short-term solution that will disable giclops spawning and remove all existing giclopes.")
-				.define("disableGiclops",true);
+				.define("disableGiclops",false);
```

Comment:
Don't change the default value of the config unless you've fixed the giclops problem